portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,30 +27,50 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+
+  List<String> comments = new ArrayList<String>();
```

Comment:
This should probably be private.

Reply:
Commits came out weirdly - done in Added @param and @return javadoc tags